src/Shellbar/Shellbar.js

```diff
@@ -514,6 +518,8 @@ Shellbar.propTypes = {
     profileMenu: PropTypes.array,
     /** Holds `searchInput` [properties](?id=component-api-searchinput--compact&viewMode=docs#properties) */
     searchInput: PropTypes.object,
+    /** Provides a size modifier for product actions only visible on small shellbar `s`, or product actions only visible on desktop screens `m`/`l`/`xl`. */
+    size: PropTypes.oneOf(['s', 'm', 'l', 'xl']),
```

Comment:
Should we also default size to 'l'?

Reply:
Added the default prop of `size` with value of `l`